ansible-galaxy: add role info and collection info examples (#6187)
tldr-pages/tldr: pages/common/ansible-galaxy.md

- Create a new role:

`ansible-galaxy init {{role_name}}`

> - Get information about a role:

Reviewer: Is information about a specific user's role? or system role?

Author: User role

Reviewer 2: Perhaps this will read better?
```suggestion
- Get information about a user role:
```

Reviewer: @papanito could you apply this change?

- Create a new role:

`ansible-galaxy init {{role_name}}`

> - Get the role id for a role name `{{username.role_name}}`:

Reviewer 3: I don't think token syntax in description lines is supported by most clients. How about just
```suggestion
- Get the role id for a role name:
```

> - Get the information for the role - or a collection if you replace `role` with `collection`:

Reviewer 2: Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
- Get the information for the role - or a collection if `role` is replaced with `collection`:
```

> `ansible-galaxy info {{username}}.{{role_name}} | grep -E 'id: [0-9]' | awk {'print $2'}`

Reviewer 3: I'm not really sure if we should include the `grep` and `awk` examples here. In general the `ansible-galaxy` page should show how the command works. If someone wants to filter the role id they can use any tools they want.

Reviewer 2: I agree with @marchersimon on this one. we generally want to avoid demonstrating the functionality of other commands here.

Reviewer: @papanito you can remove the `grep` and `awk` parts and explain what the `info` subcommand displays here.
```suggestion
`ansible-galaxy info {{username}}.{{role_name}}
```
Also, if the `.` is mandatory I think we should use the same token syntax with the first 2 examples.

Reviewer 3: @navarroaxel I suggested that above, I didn't see that the first two examples had it in the token syntax.
But why should it not be outside? We're doing that in all pages where two arguments are separated by a single character.

Reviewer: @marchersimon It's like the `ssh.md`: the `@` is outside `{{username}}@{{host}}` because it's mandatory and the `@` is not a placeholder, the same apply here.

Author: I assume then the first two examples should also be updated?

> `ansible-galaxy info YourUser.RoleName | grep -E 'id: [0-9]' | awk {'print $2'}`

Author: Yes... I will make that more clear

> `ansible-galaxy info {{username.role_name}} | grep -E 'id: [0-9]' | awk {'print $2'}`

Reviewer 3: ```suggestion
`ansible-galaxy info {{username}}.{{role_name}} | grep -E 'id: [0-9]' | awk {'print $2'}`
```
Actually, since the dot is mandatory it doesn't go in token syntax

> - Get the information for the role - or a collection if you replace `role` with `collection`:
> 
> `ansible-galaxy role info {{username}}.{{role_name}}`

Reviewer 3: How about we rephrase this a bit and split it into two examples? Also adresses @sbrl's suggestion above
```suggestion
- Get information about a role:

`ansible-galaxy role info {{username}}.{{role_name}}`

- Get information about a collection:

`ansible-galaxy collection info {{username}}.{{collection_name}}`
```
(I'm not sure if the second one is correct tough)

Author: Seems good
